### ISSUE TITLE
Something wrong with get_pin_bool

### ISSUE THREAD

**Issue by Pugnator** [Pugnator/openvsm#12]
The sample script addr.lua works weird:
The timing seems to be 2 times slower I expect. But callback function is called just in time, so nothing wrong with callback itself.

**Comment by Pugnator** [comment]
Error is in
```
STATE get_pin_state ( IDSIMPIN* pin )
{
	return pin->vtable->istate ( pin, 0 );
}
```

It should be 

```
return pin->vtable->getstate ( pin, 0 );
```

The same error affected  get_pin_bool as for some reason I didn't reused get_pin_state

**Comment by Pugnator** [comment]
Fixed in 79f1bc3a28
